Makefile

```
@@ -578,10 +582,11 @@ _expects-database.%:
 ########################################################################################
 
 e2e-tests: ## Run the end-to-end tests against the service containers.
```

```suggestion
e2e-tests: dev.up.lms+studio ## Run the end-to-end tests against the service containers.
```
How about making both this and `e2e-tests.with-shell` targets depend on `dev.up.lms+studio`? They break if those services haven't been brought up, right?